styles/wiki.nixos.org/catppuccin.user.css

```
@@ -0,0 +1,1480 @@
+    tr {
+      background-color: @surface0 !important;
+    }
+
+    th {
+      background: @overlay0 !important;
+      color: @mantle !important;
+    }
+
+    .quotebox,
+    div.thumbinner {
+      background-color: @surface0 !important;
+      border-color: @surface2 !important;
+    }
+
+    .navbox-group,
+    .infobox-label {
+      color: @text !important;
+    }
+
+    .cdx-button:enabled,
+    .cdx-text-input__input:enabled {
+      color: @text;
+      background-color: @base;
+      border-color: @surface2;
+      &:hover {
+        background-color: @mantle;
+        border-color: @text;
+        color: @text;
+      }
+    }
+
+    .vector-dropdown .vector-dropdown-content,
+    .header-container.header-chrome {
+      background-color: @mantle;
+    }
+
+    .skin-vector .uls-search {
+      border-bottom-color: @surface2;
+    }
+
+    .oo-ui-textInputWidget,
+    .oo-ui-inputWidget-input {
+      color: @text !important;
+      background-color: @surface1 !important;
+      border-color: @surface2 !important;
+    }
+
+    .oo-ui-pendingElement-pending {
+      background-color: @base;
+      background-image: linear-gradient(
+        135deg,
+        @surface0 25%,
+        transparent 25%,
+        transparent 50%,
+        @surface0 50%,
+        @surface0 75%,
+        transparent 75%,
+        transparent
+      );
+    }
+
+    .oo-ui-tagItemWidget.oo-ui-widget-disabled {
+      color: @text;
+      background-color: @overlay0;
+      text-shadow: 0 0 0 @text;
+      border-color: @surface0;
+    }
+
+    .oo-ui-buttonElement-frameless.oo-ui-widget-enabled
+      > .oo-ui-buttonElement-button,
+    .mw-mmv-credit,
+    .mw-mmv-options-dialog-header,
+    .mw-mmv-options-text-header {
+      color: @text;
+    }
+
+    .mw-mmv-options-text-body {
+      color: @subtext0;
+    }
+
+    .mw-ui-input:not(:disabled) {
+      background-color: @surface0;
+      color: @text;
+      border-color: @surface2;
+    }
+
+    .mw-ui-button {
+      color: @mantle;
+      background-color: @accent-color;
+      border-color: @accent-color;
+    }
+
+    .imbox-delete {
+      border-color: @red !important;
+      background-color: @surface0 !important;
+    }
+
+    .imbox {
+      background-color: @overlay0 !important;
+      border-color: @peach !important;
+    }
+
+    .talkheader-help {
+      background-color: @surface1 !important;
+      border-color: @green !important;
+    }
+
+    .assess,
+    .assess-NA,
+    .navbox-subgroup {
+      background: @surface0 !important;
+      border-color: @surface1 !important;
+    }
+
+    .documentation,
+    .documentation-container,
+    .documentation-metadata {
+      background-color: fade(@green, 15%) !important;
+    }
+
+    .ambox,
+    .portalborder {
+      background-color: @surface1 !important;
+      border-color: @surface2 !important;
+    }
+
+    .navbox-title {
+      color: @text !important;
+    }
+
+    .mw-highlight pre {
+      background: @surface0 !important;
```

```suggestion
      background: @mantle !important;
```

I think I prefer the contrast of this.